src/iotjs_util.c

```diff
@@ -95,39 +95,15 @@ void iotjs_buffer_release(char* buffer) {
 }
 
 void print_stacktrace() {
-#if defined(__linux__) && defined(DEBUG)
-  // TODO: support other platforms
-  const int numOfStackTrace = 100;
-  void* buffer[numOfStackTrace];
-  char command[256];
-
-  int nptrs = backtrace(buffer, numOfStackTrace);
-  char** strings = backtrace_symbols(buffer, nptrs);
-
-  if (strings == NULL) {
-    perror("backtrace_symbols");
-    exit(EXIT_FAILURE);
+#if (defined(__linux__) || defined(__APPLE__))
+  void* bt[IOTJS_BACKTRACE_LEN];
```

Comment:
@legendecas May I ask you to revert the name to buffer?

Reply:
its not a buffer. this is a storage of backtrace.

Reply:
So what's the `bt` for?